### ISSUE TITLE
Training on coco2014 using retinanet-101 just reached AP(@0.5:0.95)  of 0.302 after 16 Epochs

### ISSUE THREAD

**Issue by LiRWZ** [yhenon/pytorch-retinanet#142]
Is that normal? or I need more Epochs?

**Comment by SISTMrL** [comment]
hello, I'm sorry to disturbing you due to this is my homework, I havae a question. I encounter this question:  
No such file: '/home/mrl/Documents/retinanet/RetinaNet/datasets/COCO/images/train2017/000000171857.jpg, when run the python code: for iter_num, data in enumerate(dataloader_train), did you face the same problem, thanks!

**Comment by SISTMrL** [comment]
hello, I'm sorry to disturbing you due to this is my homework, I havae a question. I encounter this question:
No such file: '/home/mrl/Documents/retinanet/RetinaNet/datasets/COCO/images/train2017/000000171857.jpg, when run the python code: for iter_num, data in enumerate(dataloader_train), did you face the same problem,   thanks!

**Comment by LiRWZ** [comment]
sorry i have never met this erro.but i think this problem may cause by the picture (000000171857.jpg) is not in the folder.check it.if it's not there,download it.


 
---Original---

**Comment by SISTMrL** [comment]
I'm sorry to disturb you, when i see the code, i saw the retinanet.train() followed by retinanet.module.freeze_bn() in line 109 and 110 in train.py, i don't understood. I check the pytorch documentation, eval is used to model evaluation, why author use freeze_bn in train.py? thanks!

**Comment by dreamcontinue** [comment]
Hi, have you solved this problem, I have trained 40+ epoches and AP(@0.5:0.95)  is still 0.3. Will more epoches be better?

**Comment by feiyuhuahuo** [comment]
@SISTMrL  I heard that when training batch size is small (e.g. <=4), then it's not suitable to use batch normalization.
